index.html

```diff
@@ -1851,39 +1850,321 @@ <h5><code>invokeaction</code></h5>
+          mechanism for Consumers to subscribe to events emitted by a Web Thing.
+        </p>
+        <p class="note">
+          Consumers are not required to implement the 
+          <a href="https://www.w3.org/TR/eventsource/#the-eventsource-interface">
+          <code>EventSource</code>
+          JavaScript API</a> from the Server-Sent Events
+          specification in order to conform with this profile. Any programming
+          language may be used to consume an event stream.
+        </p>
+        <section id="subscribeevent">
+          <h5><code>subscribeevent</code></h5>
+          <p class="rfc2119-assertion" 
+            id="core-profile-protocol-binding-events-subscribeevent-1">
+            The URL of an <code>Event</code> resource to be used when 
+            subscribing to an event MUST be obtained from a Thing Description 
+            by locating a 
+            <a href="https://www.w3.org/TR/wot-thing-description/#form">
+            <code>Form</code></a> inside the corresponding 
+            <a href="https://www.w3.org/TR/wot-thing-description/#eventaffordance">
+            <code>EventAffordance</code></a> for which:
+          </p>
+          <ul class="rfc2119-assertion" 
+            id="core-profile-protocol-binding-events-subscribeevent-2">
+            <li>
+              After defaults have been applied, its <code>op</code> member 
+              contains the value <code>subscribeevent</code>
+            </li>
+            <li>
+              After being resolved against a base URL where applicable, the
+              <a href="https://tools.ietf.org/html/rfc3986#section-3.1">URI
+              scheme</a> [[RFC3986]] of the value of its <code>href</code>
+              member is <code>http</code> or <code>https</code>
+            </li>
+            <li>Its <code>subprotocol</code> member has a value of
+              <code>"sse"</code>
+            </li>
+          </ul>
+          <p class="rfc2119-assertion" 
+            id="core-profile-protocol-binding-events-subscribeevent-3">
+            The resolved value of the <code>href</code> member MUST then be used
+            as the URL of the <code>Event</code> resource.
+          </p>
+          <p class="rfc2119-assertion" 
+            id="core-profile-protocol-binding-events-subscribeevent-4">
+            In order to subscribe to an event, a Consumer MUST follow the 
+            Server-Sent Events [[EVENTSOURCE]] specification to open a
+            connection with the Web Thing at the URL of the <code>Event</code>
+            resource.
+          </p>
           <p>
-            Other operations under consideration include
-            <code>subscribeevent</code>, <code>unsubscribeevent</code>,
-            <code>subscribeallevents</code>, <code>unsubscribeallevents</code>,
-            <code>readpastevents</code> and <code>readallpastevents</code>.
+            This will usually involve the Consumer sending an HTTP request to
```

Comment:
This is badly worded. I was trying to avoid normative language because the normative text is provided by the Server-Sent Events specification and this is just an informative guide. Technically yes the `Accept` header could also include other formats in which case the request would be different, but to avoid ambiguity I've removed the word "usually".